vispy/visuals/tests/test_ellipse.py

```diff
@@ -19,20 +19,20 @@ def test_circle_draw():
         ellipse = visuals.Ellipse(pos=(75, 35, 0), radius=20,
                                   color=(1, 0, 0, 1))
         c.draw_visual(ellipse)
-        assert_image_equal("screenshot", 'visuals/circle1.png')
+        assert_image_approved("screenshot", 'visuals/circle1.png')
```

Comment:
Yes, the plan is to fold `assert_image_equal` into `assert_image_match` using an `image_corr` argument or similar.